zebra/if_netlink.c

```diff
@@ -310,10 +310,22 @@ static void netlink_vrf_change(struct nlmsghdr *h, struct rtattr *tb,
 	nl_table_id = *(uint32_t *)RTA_DATA(attr[IFLA_VRF_TABLE]);
 
 	if (h->nlmsg_type == RTM_NEWLINK) {
+		vrf_id_t exist_id;
+
 		if (IS_ZEBRA_DEBUG_KERNEL)
 			zlog_debug("RTM_NEWLINK for VRF %s(%u) table %u", name,
 				   ifi->ifi_index, nl_table_id);
 
+		exist_id = vrf_lookup_by_table(nl_table_id, ns_id);
+		if (exist_id != VRF_DEFAULT) {
+			vrf = vrf_lookup_by_id(exist_id);
+
+			flog_err(
+				EC_ZEBRA_VRF_MISSCONFIGURED,
```

Comment:
Not a native speaker, but shouldn't this be MISCONFIGURED?

Reply:
Yep

Reply:
fixed